synnax/pkg/hardware/rack/ontology.go

```diff
@@ -28,6 +29,18 @@ func OntologyID(k Key) ontology.ID {
 	return ontology.ID{Type: OntologyType, Key: k.String()}
 }
 
+func OntologyIDs(keys []Key) []ontology.ID {
```

Comment:
there is alot of repeated declarations of this function. Is this necessary or is there a better way to abstract this out?

Reply:
same for the other functions below

Reply:
The function itself simply casts a uuid or a int type to a ontologyID type while attaching the type of the ID ("channel", "label", etc). Do you think it's better to have the caller provide both the type and the ID instead of calling a specific type's `OntologyID` method while providing just the ID?

Reply:
I think the latter is more succinct, but it is true that that involves defining the same method in every package

Reply:
hm good point. I think the latter is clearer and the additional abstraction doesn't create much benefit here.